### PR TITLE
loader added to main page

### PR DESCRIPTION
**Pull request by gaurang444**
Please review
Fixes issue #357 Added loader to main page.
Screenshots->
loader

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <!doctype html>
 <html lang="en">
   <head>
+
```
Comment (anshumanv) [inline public/index.html]: Why are these extra lines needed?

File: public/index.html [hunk 2]
```diff
@@ -24,11 +27,21 @@
...
   <body>
+    <div class="loader"></div>
     <noscript>
       You need to enable JavaScript to run this app.
     </noscript>
     <div id="root" style="height: 100%"></div>
 
   </body>
+  <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
+<script src="http://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.2/modernizr.js"></script>
```
Comment (anshumanv) [inline public/index.html]: fix indentation

File: public/main.css [hunk 3]
```diff
@@ -19,4 +19,14 @@
...
\ No newline at end of file
+}
+.loader {
+    position: fixed;
+    left: 0px;
+    top: 0px;
+    width: 100%;
+    height: 100%;
+    z-index: 9999;
+    background: url('https://en.bloggif.com/tmp/9e7a096e0b7ed91351d8ddef3716ca43/text.gif?1512818096') 50% 50% no-repeat rgb(249,249,249);
```
Comment (anshumanv) [inline public/main.css]: can you update the loading image to be somewhat susi related?

**Comment by gaurang444** [comment]
Ok I will improve it as per the suggestions:)

**Comment by gaurang444** [comment]
Please can you suggest the appropriate image or gif for loader.

**Comment by anshumanv** [comment]
I'm not sure, let's see if others suggest something appropriate.

**Comment by gaurang444** [comment]
@anshumanv  I have fixed the indentation,package-lock.json file removed,blank lines removed.
Please review and merge pr.

File: public/index.html [hunk 4]
```diff
@@ -24,11 +24,21 @@
...
     <noscript>
       You need to enable JavaScript to run this app.
     </noscript>
     <div id="root" style="height: 100%"></div>
 
   </body>
+  <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.5.2/jquery.min.js"></script>
+  <script src="http://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.2/modernizr.js"></script>
+  <script type="text/javascript">
+    $(window).load(function() {
```
Comment (anshumanv) [inline public/index.html]: Indentation of this block is still not correct.

File: public/main.css (hunk 3 above)
Comment (anshumanv) [reply public/main.css]: The url link opens some banner thing which is not appropriate, please confirm

**Comment by anshumanv** [comment]
@gaurang444 thanks for incorporating the changes, please add a deployment link as well and squash your commits after this is approved.

I'm not a maintainer so I can't merge this, please wait for other reviewers in that case.

**Comment by ParthS007** [comment]
@gaurang444 If you use default typo and color theme, it will look better (Just a suggestion).

**Comment by Orbiter** [comment]
I believe the remote BG image https://en.bloggif.com/tmp/9e7a096e0b7ed91351d8ddef3716ca43/text.gif?1512818096 does not exist any more.
Please do not use such kind of remote links. They enable a tracking of the users activities from the outside and are a privacy issue. Always use images within the package.

**Comment by mariobehling** [comment]
Not needed.

**Pull request closed without merging**